packages/gatsby-plugin-page-creator/src/gatsby-node.ts

```diff
@@ -112,20 +124,22 @@ Please pick a path to an existing directory.`)
           })
           knownFiles.delete(removedPath)
         } catch (e) {
-          reporter.panic(
-            e.message.startsWith(`PageCreator`)
-              ? e.message
-              : `PageCreator: ${e.message}`
-          )
+          reporter.panic({
+            id: prefixId(CODES.Generic),
```

Comment:
There's something to be said about segmenting errors in file system specific ones. This could surface particular file issues that are caused by missing cross platform tests (like moving files in WSL).